src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/BurdenEstimateRepository.kt

```diff
@@ -16,7 +16,10 @@ interface BurdenEstimateRepository : Repository
                                 properties: CreateBurdenEstimateSet,
                                 uploader: String, timestamp: Instant): Int
 
+    fun getBurdenEstimateSet(groupId: String, touchstoneVersionId: String, scenarioId: String,
```

Comment:
Why not just replace the existing `getBurdenEstimateSet` method by this one? As far as I can see it's only used when populating a set, and we should be validating that the group/touchstone/scenario path is the correct one there also.